.changes/next-release/Bug Fix-e3752045-17ab-4620-ae97-438c00d24bae.json

```
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "cloudControl: Avoid redundant close of active editor when resource documents are explicitly closed"
```

in previous changelogs we called this "Resources" https://github.com/aws/aws-toolkit-vscode/blob/master/CHANGELOG.md#1320-2021-10-21

```suggestion
	"description": "Resources: Avoid redundant close of active editor"
```